pallets/subtensor/src/math.rs

```diff
@@ -807,6 +832,42 @@ mod tests {
         vector.iter().map( | x | I32F32::from_num( *x ) ).collect()
     }
 
+    #[test]
```

Comment:
I like the quantity of tests!... though am somewhat concerned what'll happen if an max values for `I32F32` are passed, because at first glance it seems like that may be a panic condition

Reply:
Added handling in case of unnormalized values, and added a function comment that specifies assumption of non-negative normalized inputs.